lib/manager/helmv3/artifacts.spec.ts

```diff
@@ -64,10 +60,14 @@ describe('manager/helmv3/artifacts', () => {
       })
     ).toBeNull();
   });
+
   it('returns null if unchanged', async () => {
-    fs.readFile.mockResolvedValueOnce(ociLockFile1 as any);
+    fs.readLocalFile.mockResolvedValueOnce(ociLockFile1 as any);
+    fs.getSiblingFileName.mockReturnValueOnce('Chart.lock');
     const execSnapshots = mockExecAll(exec);
-    fs.readFile.mockResolvedValueOnce(ociLockFile1 as any);
+    fs.readLocalFile.mockResolvedValueOnce(ociLockFile1 as any);
+    fs.privateCacheDir.mockReturnValue('');
```

Comment:
```suggestion
    fs.privateCacheDir.mockReturnValue('/tmp/renovate/cache/__renovate-private-cache');
```
This should reduce snapshot changes.
Can you add this to the other mocks too?